### DISCUSSION TITLE
[功能请求] 希望吐槽可以增加全屏显示

### DISCUSSION THREAD

**Discussion by lwp2070809** [xiaoyaocz/flutter_dmzj#109] [Ideas]
![Screenshot_20230315_182652](https://user-images.githubusercontent.com/15991672/225282091-b4556e24-0496-4466-aeb4-a82a846c135a.jpg)
希望可以实现，上划吐槽界面以全屏显示吐槽的功能。非常感谢。

**Comment by Schezuk** [comment]
+1
或者至少占屏幕70%以上，吐槽高度也压窄一些。
![image](https://user-images.githubusercontent.com/7524886/225322406-f78bb55b-8954-455c-a597-2266f3d6588f.png)

**Comment by xiaoyaocz** [comment]
https://github.com/xiaoyaocz/flutter_dmzj/releases/tag/v2.0.3-beta

**Reply by lwp2070809** [reply]
已更新，非常感谢！
